File: 3811-reverse-degree-of-a-string/reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int n=s.size();
        int sum=0;
        for(int i=0;i<n;i++){
            int reverse='z'-s[i]+1;
            sum+=(i+1)*reverse;
        }
        return sum;
    }
};

File: 3811-reverse-degree-of-a-string/reverse-degree-of-a-string.py
class Solution(object):
    def reverseDegree(self, s):
        """
        :type s: str
        :rtype: int
        """
        n=len(s)
        sum=0
        for i in range(0,n):
            reverse=ord('z')-ord(s[i])+1
            sum+=(i+1)*reverse
            
        return sum
        